app/org/sagebionetworks/bridge/models/accounts/UserSession.java

```diff
@@ -54,7 +54,6 @@ public String getInternalSessionToken() {
     public void setInternalSessionToken(String internalSessionToken) {
         this.internalSessionToken = internalSessionToken;
     }
-    @JsonIgnore
     public String getReauthToken() {
```

Comment:
IIRC the reason why we didn't include the reauth token in the session was so that we would produce it only in API calls where the user provides credentials. Would it be better to leave the JsonIgnore here and inject the reauth token into the session as part of reauth?

Reply:
Switched to storing session token and reauth token separately, and adding reauth token back to the session when it's recovered through this API's grace period.